vng_api_common/audittrails/models.py

```diff
@@ -0,0 +1,37 @@
+import uuid
+
+from django.contrib.postgres.fields import JSONField
+from django.core.serializers.json import DjangoJSONEncoder
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..descriptors import GegevensGroepType
+from .constants import AuditTrailAction
+
+
+class AuditTrail(models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=uuid.uuid4,
+        help_text="Unieke resource identifier (UUID4)"
+    )
+    bron = models.CharField(max_length=50)
+    actie = models.CharField(
+        choices=AuditTrailAction.choices,
```

Comment:
Please remove the `choices`. Although we typically expect one of these choices, it's not limited to these choices, so it needs to be just a CharField without choices.